### ISSUE TITLE
Voices not working in the latest version

### ISSUE THREAD

**Issue by BenjaTK** [mapedorr/popochiu#85]
From what I can tell, since it uses a String for the file name, the Audio singleton doesn't have the function to play based on ids anymore:
```gdscript
var vo_name := _get_vo_cue(emotion)
	if vo_name:
		A.play(vo_name, false, global_position)
```

**Comment by mapedorr** [comment]
Could be something related with the function that tries to get the name of the AudioCue to play. Let me check if I can fix it in the following hours.

**Comment by mapedorr** [comment]
@BenjaTK , you were right. You can fix it with `A[vo_name].play(false, global_position)`. Sorry for my late response. I had to run to finish a thing I'll have to show tomorrow.

**Comment by mapedorr** [comment]
Solved in 1f70d40b44446a91ac4e415d09d3088e83a40d4f
